### PR TITLE
New module mummer

### LINKED ISSUE #939: new module: mummer
I think it would be good to have a module for [mummer](http://mummer.sourceforge.net)

- [x] This module does not exist yet with the [`nf-core modules list`](https://github.com/nf-core/tools#list-modules) command
- [x] There is no [open pull request](https://github.com/nf-core/modules/pulls) for this module
- [x] There is no [open issue](https://github.com/nf-core/modules/issues) for this module
- [x] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by mjcipriano**
## PR checklist

Closes #939

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

File: modules/mummer/main.nf [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+def VERSION = '3.23'
```
Comment (lescai) [inline modules/mummer/main.nf:7]:
```suggestion

```

Comment (lescai) [inline modules/mummer/main.nf:7]: please use as much as possible a native software print of version

File: modules/mummer/main.nf [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    fi
+    mummer \\
+        $options.args \\
+        $fasta_name_ref \\
+        $fasta_name_query \\
+        > ${prefix}.coords
+
+    cat <<-END_VERSIONS > versions.yml
+    ${getProcessName(task.process)}:
+        ${getSoftwareName(task.process)}: \$( echo $VERSION )
```
Comment (lescai) [inline modules/mummer/main.nf:52]:
```suggestion
        ${getSoftwareName(task.process)}: \$( echo \$(mummer -V) )
```

Comment (lescai) [reply modules/mummer/main.nf:52]: verify if this needs to be piped with sed to clean any information other than version number

**Review by Emiller88 — APPROVED** [review]

**Review by lescai — CHANGES_REQUESTED** [review]
just a minor change to the version emission, other than that it LGTM. happy to approve once change is in

**Review by lescai — APPROVED** [review]

**Pull request merged**